packages/instantsearch.js/src/connectors/frequently-bought-together/__tests__/connectFrequentlyBoughtTogether-test.ts

```diff
@@ -40,6 +40,21 @@ describe('connectFrequentlyBoughtTogether', () => {
     );
   });
 
+  it('accepts custom parameters', () => {
+    const render = jest.fn();
+    const unmount = jest.fn();
+
+    const customFrequentlyBoughtTogether = connectFrequentlyBoughtTogether<{
+      container: string;
+    }>(render, unmount);
+    const widget = customFrequentlyBoughtTogether({
+      container: '#container',
+      objectIDs: ['1'],
+    });
+
+    expect(widget.$$type).toBe('ais.frequentlyBoughtTogether');
+  });
```

Comment:
This is only properly asserted in type checking with `tsc`, right?

Reply:
yes, we don't really have type-only tests, but tsc is asserting that there's no errors here